addons/nuxeo-csv/nuxeo-csv-core/src/test/java/org/nuxeo/ecm/csv/core/TestCSVImporterCreateMode.java

```diff
@@ -644,6 +646,49 @@ public void shouldCreateDocumentWithCustomDateformat() throws IOException, Inter
         assertEquals("2012/12/12", options.getDateFormat().format(issueDate.getTime()));
     }
 
+    /**
+     * @since 11.1
+     */
+    @Test
+    public void whenFileIsGreaterThanThresholdTotalDocsShouldBeAvailableWhenImportIsCompleted()
+            throws IOException, InterruptedException {
+        CSVImporterOptions options = new CSVImporterOptions.Builder().importMode(ImportMode.CREATE).build();
+        String importId = csvImporter.launchImport(session, "/", getCSVBlob(FILE_GREATER_THAN_THE_LENGTH_THRESHOLD),
+                options);
+        workManager.awaitCompletion(1, TimeUnit.SECONDS);
+        TransactionHelper.startTransaction();
+
+        CSVImportStatus importStatus = csvImporter.getImportStatus(importId);
+        assertNotNull(importStatus);
+        assertFalse(importStatus.isComplete());
+        assertEquals(-1, importStatus.getTotalNumberOfDocument());
+
+        workManager.awaitCompletion(10000, TimeUnit.SECONDS);
+        TransactionHelper.startTransaction();
+
+        importStatus = csvImporter.getImportStatus(importId);
+        assertTrue(importStatus.isComplete());
+        assertEquals(10999, importStatus.getTotalNumberOfDocument());
+        assertEquals(10999, importStatus.getNumberOfProcessedDocument());
+    }
+
+    /**
+     * @since 11.1
+     */
+    @Test
+    public void whenFileIsBelowTheThresholdTotalDocsShouldBeAvailableWhenImportIsRunningOrScheduled()
```

Comment:
whenFileIsBelowTheThreshold -> whenFileIsBelowThanThreshold I guess?

Reply:
the name will be replaced by `whenFileIsSmaller...` to be consistent with the other test and this way we mark a compraison => use `than`